sycl/source/backend.cpp

```diff
@@ -109,6 +140,18 @@ __SYCL_EXPORT queue make_queue(pi_native_handle NativeHandle,
   }
 }
 
+__SYCL_EXPORT queue make_queue2(pi_native_handle NativeHandle,
+                                int32_t NativeHandleDesc,
+                                const context &Context, const device *Device,
+                                bool KeepOwnership,
+                                const property_list &PropList,
+                                const async_handler &Handler, backend Backend) {
+  const auto &DeviceImpl = getSyclObjImpl(*Device);
+  return make_queue_impl2(NativeHandle, NativeHandleDesc, Context,
```

Comment:
this is the only user of `make_queue_impl2` so please jsut inline

Reply:
I wasn't able to write the conversion from DeviceImpl->getHandleRef() to  RT::PiDevice Device in the inline version. Somehow, as a function parameter the compiler does the conversion.
Can you suggest the conversion expression?

Reply:
could we have a better name other than "2" ?

Reply:
The name is internal and we have documented in several places that all the "2" versions are temporary. There is make_queue2, getNative2, piextQueueCreate2, piextQueueGetNativeHandle2, piextQueueCreateWithNativeHandle2 and so on, and they all hang together.